app/javascript/components/osbb/Tab.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Tab extends Component {
+  static propTypes = {
+    activeTab: PropTypes.string.isRequired,
+    label: PropTypes.string.isRequired,
+    onClick: PropTypes.func.isRequired,
+  };
+
+  onClick = () => {
+    const { label, onClick } = this.props;
+    onClick(label);
+  }
+
+  render() {
+    const {
+      onClick,
+      props: {
+        activeTab,
+        label,
+      },
+    } = this;
+
+    let className = 'tab-list-item';
+
+    if (activeTab === label) {
+      className += ' tab-list-active';
+    }
+
+    return (
+      <li
+        className={className}
+        onClick={onClick}
+      >
```

Comment:
can't you write in just one line?, it would not be too long

Reply:
fixed in one line